interface/patient_file/summary/labdata.php

```diff
@@ -79,390 +80,418 @@
 
```

Comment:
Hi @tywrenn 

Please help me to check this is success to compile from labdata.scss. In my environment, it seems like didn't compile.

Reply:
Where are you trying to put the SCSS? For now just ignore converting it into SCSS as i've said in a similar PR that i'll be modifying gulp and creating a folder where all these residual files will go.

Reply:
I manually copy and rename it into `labdata.css`, put the same directory.